zwave_js/DOCS.md

```diff
@@ -78,11 +82,35 @@ You can also use sites like this one to generate the required data:
 
 <https://www.random.org/cgi-bin/randbyte?nbytes=16&format=h>
 
-Ensure you keep a backup of this key. If you have to rebuild your system and
-don't have a backup of this key, you won't be able to reconnect to any securely
+Ensure you keep a backup of these key. If you have to rebuild your system and
+don't have a backup of these keys, you won't be able to reconnect to any securely
 included devices. This may mean you have to do a factory reset on those devices
 and your controller, before rebuilding your Z-Wave network.
 
+#### Option `s0_legacy_key`
+
+S0 Security Z-Wave devices require a network key before being added to the network.
+You must set the `s0_legacy_key` configuration option to use a network key before
+adding these devices.
+
+### Option `s2_access_control_key`
+
+S2 Security Z-Wave devices require three network keys before being added to the
+network. You must set this configuration option along with `s2_authenticated_key`
+and `s2_unauthenticated_key` before adding these devices.
```

Comment:
We should clarify that it's only needed to support S2 security. These devices will also work with S0 legacy.